packages/stems/src/components/SelectablePill/SelectablePill.tsx

```diff
@@ -0,0 +1,44 @@
+import { forwardRef } from 'react'
+
+import cn from 'classnames'
+
+import styles from './SelectablePill.module.css'
+import { SelectablePillProps } from './types'
+
+export const SelectablePill = forwardRef<
+  HTMLButtonElement,
+  SelectablePillProps
+>(
+  ({
+    size,
+    isSelected,
+    label,
+    icon: IconComponent,
+    onClick
+  }: SelectablePillProps) => {
+    const handleClick = () => {
+      if (!isSelected && onClick) {
```

Comment:
Did you mean to not call the click handler here if the component isn't selected?

Also wondering if there was any discussion on what the desired _developer interface_ is for Harmony components. This feels like one of those nuances where we might want to introduce a convention and make all components work the same.
In this case, that convention is that SelectablePill is "controlled" and you have to pass the selection state in from above, and also handle the clicks there.

edit: Also wondering if we even need an intermediate handler here? I think Button just passes the `onClick` handler through to the native button.

Reply:
oh wow I did not mean to do that, don't know what I was thinking lol. I think I copied some code over without thinking.
Yes SelectablePill should be controlled and we don't need an intermediate handler!